gocd/api/stage.py

```diff
@@ -0,0 +1,71 @@
+from gocd.api.endpoint import Endpoint
+
+__all__ = ['Stage']
+
+
+class Stage(Endpoint):
+    base_path = 'go/api/stages/{id}'
+
+    def __init__(self, server, pipeline_name, pipeline_counter, stage_name):
```

Comment:
I'm thinking of maybe changing the signature of this to:

```python
def __init__(self, server, pipeline_name, stage_name, pipeline_counter=None)
    pass
```

Reasons:
 - The pipeline counter is only used in one of the 3 possible calls to this Go endpoint (I've implemented cancel locally and will push to develop soon)
 - The pipeline instance call uses falsey to fetch latest instance, we would be consistent by not forcing a number in this case

What do you think?